generators/react/templates/src/main/webapp/app/entities/_entityFolder_/_entityFile_-update.tsx.ejs

```diff
@@ -149,12 +149,14 @@ export const <%= entityReactName %>Update = () => {
   }, [updateSuccess]);
 
   const saveEntity = values => {
-<%_ for (field of fields.filter(field => !field.id)) {
+<%_ for (field of fields) {
     const fieldName = field.fieldName;
 _%>
   <%_ if (field.fieldTypeTimed)  { _%>
     values.<%= fieldName %> = convertDateTimeToServer(values.<%= fieldName %>);
-  <%_ } _%>
+  <%_ } else if (field.fieldTypeNumeric || field.id && !field.autoGenerate) { _%>
+    values.<%= fieldName %> = Number(values.<%= fieldName %>);
```

Comment:
We probably should check if it's undefined.
```suggestion
    if (values.<%= fieldName %> !== undefined) {
      values.<%= fieldName %> = Number(values.<%= fieldName %>);
    }
```

Reply:
ok